src/fb5.js

```diff
@@ -234,18 +239,46 @@ function setupPageObserver() {
 
 // cleanup
 window.addEventListener("beforeunload", () => {
-  pageObserver.disconnect();
+  cleanupPageObserver();
 
   if (feedObserver !== null) {
-    feedObserver.disconnect();
-    feedObserver = null;
+    cleanupFeedObserver();
   }
 
   if (watchObserver !== null) {
-    watchObserver.disconnect();
-    watchObserver = null;
+    cleanupWatchObserver();
   }
 });
+function cleanupPageObserver() {
+  if (pageObserver === null) return;
```

Comment:
nit: I'm not a fan of `if` without brace

Reply:
changed